File: AOJ/ITP1_re/itp_1_2_d.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
  int w, h, x, y, r;
  scanf("%d %d %d %d %d", &w, &h, &x, &y, &r);
  if (0 <= x - r && x + r <= w && 0 <= y - r && y + r <= h) {
    printf("Yes\n");
  } else {
    printf("No\n");
  }
}

File: AOJ/ITP1_re/itp_1_2_d.go
package main

import "fmt"

func main() {
	var w, h, x, y, r int
	fmt.Scan(&w, &h, &x, &y, &r)
	if 0 <= x-r && x+r <= w && 0 <= y-r && y+r <= h {
		fmt.Println("Yes")
	} else {
		fmt.Println("No")
	}
}
